File: 9-mountains/submission1.cpp
//Written by Manish#9999

#include <bits/stdc++.h>
using namespace std;

int main(){
	int a[] = {9, 8, 8, 7, 9, 8, 10, 9, 11, 10, 10, 9, 9, 8, 10, 7, 11, 8, 12, 9, 11, 10, 10, 9, 9, 8, 8, 9, 7, 8, 6};
	int len = sizeof(a)/sizeof(*a);
	for(int i=0; i<len; i++){
		if (i==0 || i == len-1) continue;
		if (a[i] > a[i-1] && a[i] > a[i+1]) cout << i << " ";
	}
	return 0;
}

File: 001-evenFibonacci/submissions/submission1.py
# Written by manish#9999
# 29/01/2020

a = [2, 8]

total = 10

while True:
    next = a[-2] + 4*a[-1]
    if next > 4*10**6: break
    a += [next]
    total += next

print(total)

